bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/Car.java

```diff
@@ -40,19 +42,22 @@ public class Car {
     private boolean disableBattery = false;
     private boolean disableCockpit = false;
     private boolean disableHvac = false;
+    private boolean disableLockStatus = false;
 
     private ChargingStatus chargingStatus = ChargingStatus.UNKNOWN;
     private ChargingMode chargingMode = ChargingMode.UNKNOWN;
     private PlugStatus plugStatus = PlugStatus.UNKNOWN;
     private double hvacTargetTemperature = 20.0;
     private @Nullable Double batteryLevel;
     private @Nullable Double batteryAvailableEnergy;
+    private @Nullable ZonedDateTime batteryStatusUpdated;
     private @Nullable Integer chargingRemainingTime;
     private @Nullable Boolean hvacstatus;
     private @Nullable Double odometer;
     private @Nullable Double estimatedRange;
     private @Nullable String imageURL;
-    private @Nullable String locationUpdated;
+    private @Nullable ZonedDateTime locationUpdated;
```

Comment:
I changed string timestamps to ZonedDateTime as suggested and it works with the supplied format.